Automattic/wp-calypso
# Stats: Add 7-day highlights to the traffic page

#### Proposed Changes

<img width="1086" alt="image" src="https://user-images.githubusercontent.com/4044428/199120419-76fcc2e3-31c5-44a1-869c-b49aa123521a.png">

- Adds a 7-day highlight component above the chart in the Traffic page.
- This feature is behind a feature flag `stats/show-traffic-highlights`. Enable the feature by adding `?flags=stats/show-traffic-highlights` to the URL.

#### Testing Instructions



* Spin up this change on a live branch.
* Navigate to the Traffic page of the Stats section.
* Ensure you don't see the new highlight cards.
* Add `?flags=stats/show-traffic-highlights` and reload the page.
* Ensure you see the new highlight cards.

#### Pre-merge Checklist






Related to #69224.
